src/applications/personalization/dashboard/components/Dashboard.jsx

```diff
@@ -130,16 +151,20 @@ BenefitPaymentsAndDebt.propTypes = {
     }),
   ),
   debtsError: PropTypes.bool,
+  paymentsError: PropTypes.bool,
 };
 
 const Dashboard = ({
   fetchFullName,
   fetchMilitaryInformation,
   fetchTotalDisabilityRating,
   getDebts,
+  getPayments,
```

Comment:
Make sure you add this one to `propTypes`

Reply:
fixed